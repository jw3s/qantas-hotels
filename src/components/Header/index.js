import React from 'react';
import styled from 'styled-components';
import logo from '../../assets/qantas-logo.png';

const Wrapper = styled.header`
  padding: 20px;
  max-width: 1072px;
  margin: auto;
  margin-bottom: 40px;
`;

const Image = styled.img`
  width: 20%;
  min-width: 250px;
`;

const Header = () => (
  <Wrapper>
    <Image src={logo} alt="logo" width="20%" />
  </Wrapper>
);

export default Header;
