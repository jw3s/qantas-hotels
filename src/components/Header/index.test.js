import React from 'react';
import renderer from 'react-test-renderer';
import Header from '.';

import 'jest-styled-components';

describe('<Header />', () => {
  it('<Header /> snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
