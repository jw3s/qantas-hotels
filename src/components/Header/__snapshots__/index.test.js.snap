// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> <Header /> snapshot 1`] = `
.c0 {
  padding: 20px;
  max-width: 1072px;
  margin: auto;
  margin-bottom: 40px;
}

.c1 {
  width: 20%;
  min-width: 250px;
}

<header
  className="c0"
>
  <img
    alt="logo"
    className="c1"
    src="qantas-logo.png"
    width="20%"
  />
</header>
`;
